idx/memory/meta_tags.go

```diff
@@ -13,9 +14,15 @@ import (
 	lru "github.com/hashicorp/golang-lru"
 )
 
-// the collision avoidance window defines how many times we try to find a higher
-// slot that's free if two record hashes collide
-var collisionAvoidanceWindow = uint32(1024)
+var (
+	// the collision avoidance window defines how many times we try to find a higher
+	// slot that's free if two record hashes collide
+	collisionAvoidanceWindow = uint32(1024)
+
+	// metrics to monitor enrichment cache efficiency
+	enrichmentCacheMisses = stats.NewCounter32("idx.memory.meta_tags.enrichment_cache.misses")
+	enrichmentCacheHits   = stats.NewCounter32("idx.memory.meta_tags.enrichment_cache.hits")
```

Comment:
see the notes about `metrics2docs` in docs/CONTRIBUTING.md

Reply:
to stay consistent with other metrics (and allow for us to add other stats in the future), i think we should call these:
```
idx.memory.meta-tags.enrichment-cache.ops.miss
idx.memory.meta-tags.enrichment-cache.ops.hit
```

e.g. look at:
```
idx/memory/find_cache.go
16:	findCacheHit = stats.NewCounterRate32("idx.memory.find-cache.ops.hit")
18:	findCacheMiss = stats.NewCounterRate32("idx.memory.find-cache.ops.miss")
24:	findCacheInvalidationsReceived = stats.NewCounterRate32("idx.memory.find-cache.invalidation.recv")
26:	findCacheInvalidationsExecuted = stats.NewCounterRate32("idx.memory.find-cache.invalidation.exec")
28:	findCacheInvalidationsDropped = stats.NewCounterRate32("idx.memory.find-cache.invalidation.drop")
```